### DISCUSSION TITLE
Just a suggestion

### DISCUSSION THREAD

**Discussion by rlgordey** [thepirat000/Audit.NET#452] [Ideas]
Audit.Net Asp.Net Core Request audits fire when data is returned by the called action as near as I can tell. However, Serilog's UseSerilogRequestLogging fire after the Result has been converted to a Json string by Asp.Net Core, usually a few hundred ms later. You may want to re-consider what event you use to fire.

**Comment by thepirat000** [comment]
I haven't used serilog request logging, but do you have an idea on how this can be done from an ActionFilter and/or middleware component?
